jazz_services-handler/package.json

```diff
@@ -14,16 +14,19 @@
     "guid": "0.0.12",
     "async": "*",
     "deasync": "*",
```

Comment:
We should remove async/deasync with the move to promises

Reply:
We are using async.eachSeries for iterating the events.records

Reply:
In cooperated the comment